plugin/src/main/java/com/microsoft/java/bs/gradle/plugin/SourceSetsModelBuilder.java

```diff
@@ -76,9 +76,6 @@ public Object buildAll(String modelName, Project rootProject) {
         gradleSourceSet.setCleanTaskName(cleanTaskName);
         Set<String> taskNames = new HashSet<>();
         gradleSourceSet.setTaskNames(taskNames);
-        taskNames.add(classesTaskName);
-        taskNames.add(cleanTaskName);
-        taskNames.add(getFullTaskName(projectPath, sourceSet.getProcessResourcesTaskName()));
```

Comment:
So we are now only monitoring the compile task?

Reply:
I keep going back and forth on this.  It shouldn't matter if more are included but I think they're meaningless in general reporting of compile progress and warning/error count.
It's easy to put them back in but I think it's cleaner without them.
You can see further down that a compile task per individual language will be added.

Reply:
I vote for without them as will :)